apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-title/index.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Render site title block.
+ *
+ * @package full-site-editing
+ */
+/**
+ * Renders the site title and allows for editing in the full site editor.
+ *
+ * @param array  $attributes Block attributes.
+ * @param string $content    Block content.
+ * @return string
+ */
+function render_site_title_block( $attributes, $content ) {
+	$site_title = get_bloginfo( 'name' );
+	ob_start();
+	?>
+	<div className="full-site-editing__site_title">
+		<?php echo $site_title; ?>
```

Comment:
If we make this `<?php bloginfo( 'name' ); ?>`, then [we can ditch](https://developer.wordpress.org/reference/functions/bloginfo/) the `$site_title` variable assignment above.

Reply:
Ah so Twenty Nineteen has the following output. Which is a bit more semantic.
```
<h1 class="site-title">
     <a href="homepagelink">$site_title;</a>
</h1>
```

Reply:
In addition to the site-title class, I think we can also optionally add `wp-block-a8c-site-title`

Reply:
> In addition to the site-title class...

Those are just for the editor view aren't they, not the front-end output?

Reply:
Publish/Edit Preview should match if possible, there's not much point making a div here and then styling it to look like an h1 when we can simply use the h1